src/data/faq.json

```diff
@@ -195,6 +195,15 @@
                                 "textblock": [
                                     "Even with an older version of the CWA, you can still trigger a warning until April 30, 2023. After that, however, the warning of others will no longer be possible."
                                 ]
+                            },
+                            {
+                                "title": "Why do I get a notification from my iPhone that exposure notifications have been disabled?",
+                                "anchor": "ramp_down_iOS",
+                                "active": false,
+                                "textblock": [
+                                    "If you have iOS 16.4 or later installed on your iPhone, you will receive a notification on June 1, 2023 informing you that exposure notifications have been disabled. This notification is not issued by the Corona-Warn-App, but directly by the operating system.",
+                                    "<b>Important:</b> As early as May 1, 2023, the Corona-Warn-App no longer performs exposure checks. If no other country's contact tracking app is installed that uses your operating system's ENF (Exposure Notification Framework), exposure logging by the operating system will no longer be active and random IDs will not be exchanged with other devices via the ENF."

```

Comment:
"As early as" => "Since"

"As early as" can mean earlier than you expected.
"Since" is more neutral

Reply:
thanks @MikeMcC399, will fix!